src/main/resources/atlassian-plugin.xml

```diff
@@ -39,9 +39,9 @@
 			<parameter name="signerGroups" multiple="false" type="string" required="false"/>
 			<parameter name="inheritSigners" multiple="false" type="enum" required="true" default="none">
```

Comment:
Is the change in the values of inheritSigners backwards compatible? 
What happens to the macro instances that were saved with "writers only" in a previous version of the plugin?

Reply:
You are correct in the implication that this change would break backwards compatibility. Furthermore, this PR did not change the .java files that rely upon the string values of inheritSigners. Fortunately, it seems that .properties files will accept keys that contain spaces if those spaces are backslash-escaped. I've added a new commit accordingly.